packages/lit-element/src/index.ts

```diff
@@ -0,0 +1,10 @@
+export * from '@lit/reactive-element';
+export * from 'lit-html';
+export * from './lit-element.js';
+
```

Comment:
Wasn't the idea to make this backward-compatible w.r.t. the exports? I guess that just means adding `export * from './decorators.js'`? That'll help with the number of g3 changes needed also.